esp32/modules/easyrtc.py

```diff
@@ -7,8 +7,11 @@
 import machine, time
 
 # Functions
-def string(date=False, time=True):
-    [year, month, mday, wday, hour, min, sec, usec] = machine.RTC().datetime()
+def string(print_date=False, print_time=True, timestamp = -1):
```

Comment:
nice naming improvement, though not strictly speaking backwards compatible. can we be sure no-one was using the old names?